src/messages/initialize.cc

```diff
@@ -86,12 +86,15 @@ struct InitializeHandler : BaseMessageHandler<Ipc_InitializeRequest> {
 
       // Ensure there is a resource directory.
       if (config->resourceDirectory.empty()) {
```

Comment:
This confused me. For my situation, the emacs plugin always points `resource-dir` to the working directory in initialization options and prevent the usage of the default one. It also seems that ".." need not to be concerned on my mac or linux.

Reply:
The emacs plugin needs to change accordingly (as I have done in my local cquery.el, but not in this pull request). This pull request is intented to give a resource dir that just works most of the time so users need not worry about setting it in Emacs/VS Code etc.